components/x-teaser/src/RelativeTime.jsx

```diff
@@ -15,17 +15,17 @@ const displayTime = (date) => {
 	return `${hours} ${suffix}`
 }
 
-export default ({ publishedDate, firstPublishedDate }) => {
+export default ({ publishedDate, firstPublishedDate, override4hLimit = false }) => {
 	const relativeDate = getRelativeDate(publishedDate)
 	const status = getStatus(publishedDate, firstPublishedDate)
 
-	return isRecent(relativeDate) ? (
+	return override4hLimit === true || isRecent(relativeDate) ? (
```

Comment:
`isRecent` function here abstracts away what recent is. It is 4 hours now, but can change in the future. If that happens your override variable name would need to be updated too.

`isRecent` is using currently a constant but it might make sense to pass an optional alternative.